lib/iris/cube.py

```diff
@@ -3055,7 +2983,32 @@ def _order(array):
 
         return cube_xml_element
 
-    def copy(self, data=None):
+    def replace(self, data, dtype=None, fill_value=None):
```

Comment:
I think `cube.replace_data(...)` would be much clearer than `cube.replace(x, y, z)`